addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CodeInjectionScanRule.java

```diff
@@ -248,64 +227,48 @@ private boolean testPhpInjection(String paramName) {
      * @see #ASP_PAYLOADS
      */
     private boolean testAspInjection(String paramName) {
-        int bignum1 = 100000 + (int) (RAND.nextFloat() * (999999 - 1000000 + 1));
-        int bignum2 = 100000 + (int) (RAND.nextFloat() * (999999 - 1000000 + 1));
+        int bignum1 = Math.abs(RAND.nextInt(Integer.MAX_VALUE));
+        int bignum2 = Math.abs(RAND.nextInt(Integer.MAX_VALUE));
```

Comment:
`abs` is not required when using limit. Maybe the value should be smaller (more like it was before).

Reply:
Set local MAX_VALUE constant 999,999. Updated UnitTest to zero or one thousands 'portion' in the regex (? vs *).

Reply:
Sounds good.